apistructs/pipeline_task.go

```diff
@@ -23,7 +23,9 @@ import (
 
 const (
 	// TerminusDefineTag add this tag env to container for collecting logs
-	TerminusDefineTag = "TERMINUS_DEFINE_TAG"
+	TerminusDefineTag            = "TERMINUS_DEFINE_TAG"
+	PipelineTaskMaxRetryLimit    = 100
```

Comment:
How much time will the normal result of 100 retry times cost?
Please compare to 24hours.